docs/source/python_api/mlflow.keras.rst

```diff
@@ -1,7 +0,0 @@
-mlflow.keras
-==============
-
-.. automodule:: mlflow.keras
-    :members:
-    :undoc-members:
-    :show-inheritance:
```

Comment:
Should we keep this file since we still have `mlflow.keras` or can we drop it?

Reply:
We can drop it